sopel/config/types.py

```diff
@@ -468,9 +472,9 @@ class SpamSection(StaticSection):
         **About:** backward compatibility with comma-separated values.
 
         A :class:`ListAttribute` option allows to write, on a single line,
```

Comment:
"initially allowed writing, on a single line,"?

Sorry I can't make this one into a suggestion unified with the changes below & nicely rewrapped for you; GitHub has blinders on and can only see those deleted lines in the middle, so it won't let me. 😑

Reply:
Hm. You can still write this on a single line, even in Sopel 8.

Reply:
Technically yes, but it's going to generate a warning if you do now. Maybe this isn't the exact phrasing to use, but I would really like to avoid use of present tense when talking about _deprecated_ support for comma-as-delimiter.